composites/OnboardingWizard/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import YoastSeoIcon from "../basic/YoastSeoIcon";
+
+const Header = () => {
+	return (
+		<header className="yoast-wizard--header">
+			<YoastSeoIcon height={56} width={56}/>
+			<h1>Yoast SEO for WordPress installation wizard</h1>
```

Comment:
this should be translatable, except "Yoast SEO for WordPress" (we never translate plugin names for branding reasons)